client/homebrew/brewRenderer/toolBar/toolBar.jsx

```diff
@@ -37,10 +39,67 @@ const ToolBar = ({ onZoomChange, currentPage, onPageChange, totalPages })=>{
 		setPageNum(pageNumber);
 	};
 
+
+	const calculateZoom = (mode)=>{
```

Comment:
I think this should be renamed to `handleZoomFit` or `handleZoomFitButtons` or something.

Reply:
I'm not sure I agree-- when it is called, it is `onClick={()=>handleZoomButton(calculateZoom('fit'))}`, so it seems weird there were be a handler inside a handler.  This function isn't changing anything on the page or specifically reacting to a button being pressed, it's just doing a calculation which is then used as an input to a handler function.  And it takes 'fit' or 'fill' as it's arguments, so it's pretty much right there.  

Maybe `calculateChange`?  

I'm going to let this simmer a bit, and maybe when addressing the other comments something better will come up.

Reply:
i changed it to calculateChange, in combo with addressing another comment